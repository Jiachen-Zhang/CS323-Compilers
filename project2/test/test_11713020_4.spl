int a()
{
  int test_11713020_4 = 1;
  return 0;
}
int test_11713020_4()
{
  return 0;
}