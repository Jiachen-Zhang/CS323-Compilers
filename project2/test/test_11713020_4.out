Error type 4 at Line 6: redefine function: test_11713020_4
